src/transformation/visitors/loops/for-of.ts

```diff
@@ -88,6 +88,7 @@ function transformForOfLuaIteratorStatement(
             // Variables NOT declared in for loop - catch iterator values in temps and assign
             // for ____value0 in ${iterable} do
             //     ${initializer} = ____value0
+            // eslint-disable-next-line no-lonely-if
```

Comment:
This rule seems a little awkward. I think this is very much a case-by-case basis and would prefer not to have a few eslint-disables as possible.

Reply:
I guess that's true, though I don't think I had any issues with it in years of being in my config. I have disabled this rule and also fixed this case, since it actually turned out to look neater with the logic flow here